lib/MTT/Test/Build/Trivial.pm

```diff
@@ -56,6 +58,12 @@ sub Build {
                        "trivial_tests_cflags");
     my $fflags = Value($ini, $config->{full_section_name}, 
                        "trivial_tests_fflags");
+    my $f77flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f77flags");
+    my $f90flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f90flags");
+    my $f08flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f08flags");
```

Comment:
Might as well change them while we're mucking around in here...

Sorry; you're getting caught up in my type-A personality, and the fact that I'm on the Fortran WG in the MPI Forum (i.e., I'm on a crusade to fix everyone's nomenclature, because the use of "F77", "F90", and "F08" is just wrong.  And yes, even Past Jeff got it wrong in this MTT code... :smile: ).

If you want to just commit this one update for the new flags, I'll be happy to do the 2nd fix of updating the rest of the $mpif77 / $mpif90 / $mpif08 stuff (I didn't look in the git logs, but it was almost certainly me who wrote that stuff in the first place!).